src/snapshot-editor/src/edit_vmstate.rs

```diff
@@ -0,0 +1,156 @@
+// Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use std::path::PathBuf;
+
+use clap::Subcommand;
+use clap_num::maybe_hex;
+use vmm::arch::aarch64::regs::Aarch64RegisterVec;
+use vmm::persist::MicrovmState;
+
+use crate::utils::{open_vmstate, save_vmstate, UtilsError};
+
+#[derive(Debug, thiserror::Error)]
+pub enum EditVmStateError {
+    #[error("{0}")]
+    Utils(#[from] UtilsError),
+}
+
+#[derive(Debug, Subcommand)]
+pub enum EditVmStateSubCommand {
+    /// Remove registers from vcpu states.
+    #[cfg(target_arch = "aarch64")]
+    RemoveRegs {
+        /// Set of regs to remove.
```

Comment:
It'd be nice to mention what these `u64` values actually are. Are they indexes, some other name, or key?

Reply:
Added another comment